### PR TITLE
Middle section of mobile version.

### PR DESCRIPTION
**Pull request by jorgegoco**
-Added image files
-Extended versions for html and css files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,7 +30,126 @@ <h1 class="black-text-header">Hey there. I’m<br> Jorge<br><span>I’m a softwa
...
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <h2 class="about-me-header">About me</h2>
+    <p class="about-me-paragraph-mobile"> I can help you build a product,<br>feature or website Look through<br>some of
+      my work and
+      experience! If<br>you like what you see and have a<br>project you need coded,<br>don’t hesitate to contact me.</p>
+    <button class="about-me-button">Get My Resume</button>
```
Comment (cynthiainga) [inline index.html]: - [ ] Good job so far by implementing the about section but I think this part is a part of the about section, please kindly add it to the about `section` tag 😁😁✌️

File: index.html [hunk 2]
```diff
@@ -31,7 +30,126 @@ <h1 class="black-text-header">Hey there. I’m<br> Jorge<br><span>I’m a softwa
...
+          <li class="skill-element">GitHub</li>
+          <li class="skill-element">Codepen</li>
+          <li class="skill-element">Gitlab</li>
+          <li class="skill-element terminal">Terminal</li>
+        </ul>
+      </div>
+    </section>
   </main>
+  <img class="about-me-first-picture" src="./assets/images/about_me_first_picture.svg">
+  <img class="about-me-second-picture" src="./assets/images/about_me_second_picture.svg">
```
Comment (cynthiainga) [inline index.html]: - [ ] The section has the proper background image (decorations are always backgrounded images, not `img` tags), please kindly fix it 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @jorgegoco, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Used correct Github flow ✔️
- [ ] Nice design 👌 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @jorgegoco , 

Good work implementing the requested changes and excellent work. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**